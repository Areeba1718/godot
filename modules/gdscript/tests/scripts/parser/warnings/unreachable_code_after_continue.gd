func test():
    var x := 10
    match x:
        0:
            continue
            print_debug("Error: print after continue")
        _:
            continue
            print_debug("Error: print after continue")
