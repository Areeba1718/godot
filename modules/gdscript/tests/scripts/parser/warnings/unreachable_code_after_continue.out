GDTEST_OK
>> WARNING
>> Line: 6
>> UNREACHABLE_CODE
>> Unreachable code (statement after return or continue) in function 'test()'.
>> WARNING
>> Line: 9
>> UNREACHABLE_CODE
>> Unreachable code (statement after return or continue) in function 'test()'.
